src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4821onevariableperline/InputOneVariablePerDeclaration.java

```diff
@@ -1,100 +1,101 @@
-        }
-        // second definition is wrapped
-        // line of VARIABLE_DEF is not the same as first line of the definition
-        java.lang.String string; java.lang.String
-        // violation above 'Only one variable definition per line allowed.'
-            strings[];
-        //both definitions are wrapped
-        java.lang. // violation 'Only one variable definition per line allowed.'
-            String string1; java.lang.String
-                strings1[];
-
-        void method2() {
-            for (int i=0, j=0; i < 10; i++, j--) { //ok
-            }
-
-            for(int i=0; i<4;i++) {
-
-            }
-        }
-        Inner anon = new Inner()
-        {
-            int i, j; // violation 'Each variable declaration must be in its own statement.'
-            int i1; int j1; // violation 'Only one variable definition per line allowed.'
-
-            void method1() {
+    Inner anon =
+            new Inner() {
+              int abc, pqr; // violation 'Each variable declaration must be in its own statement.'
+              int i1; int j1; // violation 'Only one variable definition per line allowed.'
+
+              void method1() {
                 String str, str1;
                 // violation above 'Each variable declaration must be in its own statement.'
                 java.lang.Object obj; Object obj1;
                 // violation above 'Only one variable definition per line allowed.'
-            }
-            // second definition is wrapped
-            // line of VARIABLE_DEF is not the same as first line of the definition
-            java.lang.String string; java.lang.String
-            // violation above 'Only one variable definition per line allowed.'
-                strings[];
-            //both definitions are wrapped
-            java.lang. // violation 'Only one variable definition per line allowed.'
-                String string1; java.lang.String
-                    strings1[];
-
-            void method2() {
-                for (int i=0, j=0; i < 10; i++, j--) { //ok
+              }
+
+              // second definition is wrapped
+              // line of VARIABLE_DEF is not the same as first line of the definition
+              // violation below 'Only one variable definition per line allowed.'
+              java.lang.String string; java.lang.String[]
+                  strings;
+              // both definitions are wrapped
+              // violation below 'Only one variable definition per line allowed.'
+              java.lang
+                  .String string1; java.lang.String[]
+                          strings1;
+
+              void method2() {
+                for (int i = 0, j = 0; i < 10; i++, j--) { // ok
                 }
 
-                for(int i=0; i<4;i++) {
-
-                }
-            }
-        };
-    }
-}
+                for (int i = 0; i < 4; i++) {}
+              }
+            };
+  }
 
-class Suppress {
+  class Suppress {
```

Comment:
Non top level class is moved to Inner class.
Interesting automatic decision., bit ok for us.

Reply:
had to move it inside the public class otherwise it would complain about `Top-level class should be in its own file` when we add usage of `verifyWithWholeConfig`